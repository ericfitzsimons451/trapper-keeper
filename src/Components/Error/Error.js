import React from 'react'
import './Error.scss'

export const Error = () => {
    return (
        <div className="error">
            <h2>Error 404  Page Not Found</h2>
            <img src='http://pixelartmaker.com/art/e98ded70b9e92e4.png' alt='dumpster fire' className='dumpster'/>
        </div>
    )
}

export default Error