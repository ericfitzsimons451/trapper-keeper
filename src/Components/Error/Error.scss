.error {
    margin-top: 15vh;
}

.dumpster {
    height: 40vh;
}