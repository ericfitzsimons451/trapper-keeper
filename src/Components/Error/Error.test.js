import React from 'react'
import { Error } from './Error'
import { shallow } from 'enzyme'

describe('Error', () => {
    it('should match the snapshot', () => {
        let wrapper = shallow(<Error />)
        expect(wrapper).toMatchSnapshot()
    })
})