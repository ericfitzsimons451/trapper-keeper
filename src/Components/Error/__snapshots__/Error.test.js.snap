// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Error should match the snapshot 1`] = `
<div
  className="error"
>
  <h2>
    Error 404  Page Not Found
  </h2>
  <img
    alt="dumpster fire"
    className="dumpster"
    src="http://pixelartmaker.com/art/e98ded70b9e92e4.png"
  />
</div>
`;
